source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -867,3 +1156,299 @@ The XML excerpt below is an example of a compatible XML data feed.
   </xs:element>
 </xs:schema>
 ~~~
+
+### XSD with Size Variations
```

Comment:
I am not sure how to validate that this is a valid XSD and it produces the XML we want.

Reply:
I updated the new xsd with size variations. Now it's a valid xsd and generates a valid xml if you try to transform it.

Reply:
General note: I think that the old xsd it's not a valid XSD with simple xml it's not valid. We have to fix it?

Reply:
I believe we should fix it but we can discuss about it